site/content/docs/projects/overview.md

```diff
@@ -0,0 +1,13 @@
+### Overview
+
+The deployment tool examines your .NET project and generates a CDK deployment project in your workspace. Deployment tool then uses this deployment project to provision the necessary infrastructure and deploy your application.
```

Comment:
The projects are not generated in the workspace unless they are generating a deployment project. They are generated under a special folder in the user directory. I don't think the location is relevant here but I don't want user thinking the project is generated in their project's workspace.

Reply:
Thinking about this more I assume this is meant to be a section on deployment projects but this page reads confusing because it also talks about the CDK project that we create under the covers. For example in the sentence below it says "For simple deployments where you don’t want to further customize resources, you can feel free to ignore the CDK project." which makes no sense if we are talking about deployment projects.

Reply:
Can we discuss this?